renderdoc/driver/vulkan/vk_counters.cpp

```diff
@@ -134,7 +134,9 @@ void VulkanReplay::convertKhrCounterResult(CounterResult &rdcResult,
   if(khrUnit == VK_PERFORMANCE_COUNTER_UNIT_NANOSECONDS_KHR)
   {
     RDCASSERT(type == CompType::Double);
-    rdcResult.value.d /= 1000.0 * 1000.0 * 1000.0;
+    RDCASSERT((khrStorage != VK_PERFORMANCE_COUNTER_STORAGE_FLOAT32_KHR) &&
+        (khrStorage != VK_PERFORMANCE_COUNTER_STORAGE_FLOAT64_KHR));
```

Comment:
I don't think we can assume that the storage isn't floating point (though it would be a little odd, doubles do have sufficient precision for it). It would be better here to check if the storage is in `u64` or `d` then do the divide as appropriate.